Earthfile

```
@@ -11,7 +11,7 @@ src-files:
   COPY parsers ./parsers
   COPY ./config+src-files/* ./config
   COPY scripts ./scripts
-  COPY web/public/locales/en.json ./web/public/locales/en.json
+  COPY web/src/locales/en.json ./web/public/locales/en.json
   COPY __init__.py ./__init__.py
```

I think we need to update the `dest` path as well, and update it everywhere it's used in monorepo - otherwise there will be a discrepancy between running things on your local machine vs on CI :)

```suggestion
  COPY web/src/locales/en.json ./web/src/locales/en.json
  COPY __init__.py ./__init__.py
```

---

Do de even need to copy them then? Don't we copy the entire src folder?

My thinking here was that we could merge this without impacting other deployments and then point them to the new file location if needed. But as you say it wouldn't match local development.

---

Oh god 😅

---

Which is why I thought it might be better to first split it out, and then afterwards do this change 🙈

---

Makes sense, but slitting it out would require other changes as well...

Could we just not leave a copy in the old location for now (maybe with just the zone names) and then migrate it?

---

I guess we could, then we should remove anything but zone names to avoid confusing contributors :)

---

And also make sure we update documentation in various places

---

I'll update the Wiki after this has been approved (but right before I merge it) so we don't cause a mismatch for any longer period of time.